models/user.js

```diff
@@ -19,12 +25,12 @@ schema.set('toJSON', {
   },
 });
 
-schema.methods.validatePassword = function(pwd) {
+schema.methods.validatePassword = function (pwd) {
```

Comment:
Please make sure you've got prettier installed in VSCode.

Set VSCode to "autoformat on save"

In VSCode settings, change the following from the defaults in the Prettier settings:
- `Prettier: Single Quote`: `true`
- `Prettier: Trailing Comma`: `es5`

This will prevent randomly changing one line here or there.